rules/group.go

```diff
@@ -601,14 +606,27 @@ func (g *Group) Eval(ctx context.Context, ts time.Time) {
 	g.cleanupStaleSeries(ctx, ts)
 }
 
+func (g *Group) RuleQueryOffset() time.Duration {
```

Comment:
Same here, I'd probably just call it `QueryOffset()` in this context.

Reply:
Done in https://github.com/prometheus/prometheus/pull/14061/commits/ec0cefb94e311441ea353a4a0b0f0af7b5fb96dc